docusaurus.config.js

```diff
@@ -110,18 +110,109 @@ const config = {
         hideOnScroll: true,
         items: [
           {
-            type: "docSidebar",
-            sidebarId: "docSidebar",
+            type: "dropdown",
+            label: "Product",
             position: "left",
-            label: "Docs",
-            to: "/docs/getting-started",
+            items: [
+              {
+                label: "White Paper",
+                to: "/whitepaper",
+              },
+              {
+                label: "vs MinIO",
+                to: "/blog/comparisons/computer-vision/iot/performance-comparison-reductstore-vs-minio",
+              },
+              {
+                label: "vs MinIO & InfluxDB",
+                to: "/blog/comparison/iot/reductstore-benchmark",
+              },
+              {
+                label: "vs TimescaleDB",
+                to: "/blog/comparisons/iot/reductstore-vs-timescaledb",
+              },
+              {
+                label: "vs MongoDB",
+                to: "/blog/comparisons/iot/reductstore-vs-mongodb",
+              },
+              {
+                label: "Schedule a Technical Review",
+                to: "/contact?subject=TechnicalReview",
+              },
+            ],
+          },
+          {
+            type: "dropdown",
```

Comment:
Maybe we can add it  when we will have use case pages? We discussed  them like blog posts but I believe they should have different structure.

Reply:
@atimin I agree, although, don't you think it's better than nothing while waiting for the use cases?

Reply:
Okay, let's keep it and focus on the use cases.